docs/source/support/component-statuses.rst

```diff
@@ -0,0 +1,329 @@
+     - Stable
+     -
+   * - ``@daml/types`` :doc:`library and its public API </app-dev/bindings-ts/daml-types>`
+     - Stable
+     -
+   * - ``@daml/ledger`` :doc:`library and its public API </app-dev/bindings-ts/daml-ledger>`
+     - Stable
+     -
+   * - ``@daml/react`` :doc:`library and its public API </app-dev/bindings-ts/daml-react>`
+     - Stable
+     -
+   * - **DAML Libraries**
+     -
+     -
+   * - :doc:`The DAML Standard Library </daml/stdlib/index>`
+     - Stable
+     -
+   * - :doc:`The DAML Script Library </daml-script/api/index>`
+     - Stable
+     -
+   
+.. _dev-tools:
+
+Developer Tools
+---------------
+
+.. list-table::
+   :widths: 80 10 10
+   :header-rows: 1
+
+   * - Component / Feature
+     - Status
+     - Deprecated on
+   * - **SDK**
+     -
+     -
+   * - Windows SDK (installer_)
+     - Stable
+     -
+   * - :ref:`Mac SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :ref:`Linux SDK <mac-linux-sdk>`
+     - Stable
+     -
+   * - :doc:`/tools/assistant` with top level commands
+
+       - ``--help``
+       - ``version``
+       - ``install``
+       - ``uninstall``
+     - Stable
+     -
+   * - ``daml start`` helper command and associated CLI (``daml start --help``)
+     - Stable
+     - 
+   * - ``daml deploy`` :doc:`helper command and associated CLI </deploy/generic_ledger>` (``daml deploy --help``)
+     -
+     -
+   * - **DAML Projects**
+     -
+     -
+   * - ``daml.yaml`` project specification
+     - Stable
+     - 
+   * - Assistant commands ``new``, ``create-daml-app``, and ``init``. Note that the templates created by ``daml new`` and ``create-daml-app`` are considered example code, and are not covered by :ref:`semantic versioning <versioning>`.
+     - Stable
+     -
+   * - **DAML Studio**
+     -
+     -
+   * - VSCode Extension
+     - Stable
+     -
+   * - ``daml studio`` assistant command
+     - Stable
+     -
+   * - **Code Generation**
+     -
+     -
+   * - ``daml codegen`` assistant commands
+     - See :ref:`libraries`.
+     -
+   * - **Dev-Time Runtime Components**
+     -
+     -
+   * - ``daml json-api`, ``daml trigger``, ``daml trigger-service``, and ``daml extractor`` assistant commands.
+     - See :ref:`runtime-components`.
+     -
+   * - **Sandbox Development Ledger**
+     -
+     -
+   * - ``daml sandbox`` assistant command and documented CLI under ``daml sandbox --help``.
+     - Stable
+     -
+   * - DAML Sandbox in Memory (ie without the ``--sql-backend-jdbcurl`` flag)
+     - Stable
+     -
+   * - DAML Sandbox on Postgres (iw with the ``--sql-backend-jdbcurl`` flag)
+     - Stable
+     -
+   * - DAML Sandbox Classic and associated CLIs ``daml sandbox-classic``, ``daml start --sandbox-classic``
+     - Stable
+     - 2020-04-01
+   * - **DAML Compiler**
+     -
+     -
+   * - ``daml build`` CLI
+     - Stable
+     -
+   * - ``daml damlc`` CLI
+     - Stable
+     -
+   * - DAML Language Server (``daml damlc ide``)
+     - Stable
```

Comment:
The language server is stable, not sure we want to brand the CLI stable since we don’t claim to support this outside of vscode.

Reply:
Some people use it for their own IDEs in emacs or similar. I'm fine either way. Your call.

Reply:
So far, we told people that this might work but we only support vscode so supporting the CLI seems somewhat contradictory to that. @hurryabit should probably make the call here.

Reply:
I agree with @cocreature. I'd prefer to not mark `daml damlc ide` as stable as that would increase our surface API area without adding significant value.

Reply:
Marked as Labs, thanks.